.ci/utils/use-esy.yml

```diff
@@ -0,0 +1,6 @@
+# steps to install esy globally
+
+steps:
+  # Ensure esy is able to pull in submodules (introduced in 0.6.0)
+- script: 'npm install -g esy --unsafe-perm'
```

Comment:
Would it be possible to standardize and just use yarn?

Reply:
I'll make the change

Reply:
Can we ignore this for now? I don't want to touch the CI setup right now

Reply:
Okay. Let's leave a TODO in here however (and/or an issue)